plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -168,23 +170,27 @@ public Optional<JdbcTableHandle> getTableHandle(ConnectorSession session, Schema
     {
```

Comment:
This was overriden because of some limitation in druid. I tried removing it and thats when I noticed the javadoc

 /**
     * Overridden since the {@link BaseJdbcClient#getTables(Connection, Optional, Optional)}
     * method uses character escaping that doesn't work well with Druid's Avatica handler.
     * Unfortunately, because we can't escape search characters like '_' and '%", this call
     * ends up retrieving metadata for all tables that match the search
     * pattern. For ex - LIKE some_table matches somertable, somextable and some_table.
     *
     * See {@link DruidJdbcClient#getTableHandle(ConnectorSession, SchemaTableName)} to look at
     * how tables are filtered.
     */